ecs-fargate-service/ecs-fargate-variables.tf

```diff
@@ -0,0 +1,252 @@
+variable "application" {
+  description = "Application name"
+  type        = string
+}
+
+variable "cluster_name" {
+  description = "the ECS cluster name"
+  type        = string
+}
+
+variable "cluster_arn" {
+  description = "Cluster ARN"
+  type        = string
+}
+
+variable "log_group_arn" {
+  description = "Log group arn"
+  type        = string
+}
+
+variable "vpc_id" {
+  description = "The ID of the VPC"
+  type        = string
+}
+
+variable "vpc_id_tg" {
+  description = "The ID of the VPC where TG should be created"
+  type        = string
+  default     = ""
+}
+
+variable "account_id" {
+  description = "The account ID"
+  type        = string
+}
+
+variable "name_prefix" {
+  description = "A prefix used for naming resources."
+  type        = string
+}
+
+variable "task_container_port" {
+  description = "Port that the container exposes."
+  type        = number
+  default     = 8080
+}
+
+variable "task_container_monitoring_port" {
+  description = "Port that the container exposes for monitoring"
+  type        = number
+  default     = 0
```

Comment:
No sure what happens if default value is used?  Isnt it like the system tries to find random suitable port?

Reply:
0 means that no additional port is exposed for monitoring. Fixed also the description.